### Question 1.2. (5pts) 
What are the assumptions that the Jukes-Cantor
Model makes? What would be the Jukes-Cantor distance between two
sequences, where the first has a frame shift mutation (indel) but
otherwise is identical? For example, assume the following genomic
sequences:

- S1: ATTCGAAA...**TCAAAATGCA**...ATTGDSAAAA
- S2: ATTCGAAA...**CAAAATGCAT**...ATTGDSAAAA

Only the sequences in bold are given (second sequence identical to
the first but shifted one position to the left). Calculate the JC
distance.

Four assumptions:

- The occurrence of each nucleotide base in the sequence is equally probable.
- The rate of substitution is uniform across all bases.
- The substitution process is random and independent across sites.
- The sequences undergo separate evolution without any occurrence of recombination or gene conversion events.

In this situation,  there is only one difference between the sequences, which is the insertion or deletion event. Therefore, we can say that the Jukes-Cantor distance between these two sequences is 1 substitution per site

**Comparing S1 to S2**

`p = 1/28 = 0.0357`

`d = -3/4 * ln(1 - 4/3 * 0.0357) = 0.0374`

Distance(S1, S2) = `0.0374 * 1/28 = 0.00133714285714`